### ISSUE TITLE
Fix module path in go.mod

### ISSUE THREAD

**Issue by tommyknows** [profclems/glab#189]
**Description**

The module path in the `go.mod` file is listed as `glab`, although to `go get` it, the correct path would be `github.com/profclems/glab`.

This makes it impossible to run `go get github.com/profclems/glab`.

**Expected Behavior**

```
go get github.com/profclems/glab
```
runs without an error, building the binary.

**Actual Behavior**
```
go: downloading github.com/profclems/glab v1.10.0
go: github.com/profclems/glab upgrade => v1.10.0
go get: github.com/profclems/glab@v1.10.0: parsing go.mod:
        module declares its path as: glab
                but was required as: github.com/profclems/glab
```
**Possible Fix**
Change the first line in the `go.mod` file from `module glab` to `module github.com/profclems/glab`.

Or is there a reason why this is the way it is?

**Comment by profclems** [comment]
Thanks @tommyknows 
For most CLIs built using cobra, you're likely to encounter errors using `go get` to clone and build the binary, unless you're reusing packages in the `pkg/` folder. `glab` provides no publicly usable packages yet.
Check https://github.com/profclems/glab#building-from-source if you want to build from source.

However, I have also thought of changing the module path in case we add usable packages to `glab` and also, it is very common for Go projects to be named by their GitHub path.

**Comment by tommyknows** [comment]
> For most CLIs built using cobra, you're likely to encounter errors using go get to clone and build the binary, unless you're reusing packages in the pkg/ folder. 

Not quite sure if I understand correctly, but it is possible to just `cd` into `cmd/glab` and run `go build .` in there. Because of that, `go get github.com/profclems/glab/cmd/glab` would install the binary on the system too (unless you're running this command in a module-folder).
I've written a [wrapper](https://github.com/tommyknows/packa) for this, which effectively runs `go get [URL]@[VERSION]` to build & manage Go programs / binaries.

I don't think that anything else needs to be done except fixing the module path.

**Comment by profclems** [comment]
Resolved by #190

**Comment by tommyknows** [comment]
very nice, thanks for the quick response! 
I can confirm that it works as expected:
```
$> go get github.com/profclems/glab/cmd/glab@trunk
go: downloading github.com/profclems/glab v1.10.1-0.20200902122033-b8e5b6716e6c
go: found github.com/profclems/glab/cmd/glab in github.com/profclems/glab v1.10.1-0.20200902122033-b8e5b6716e6c
go: downloading github.com/gookit/color v1.2.9
go: downloading google.golang.org/protobuf v1.23.0
$> glab -h
GLab is an open source Gitlab Cli tool bringing GitLab to your command line
...
```
